k8s/dockerfiles/analytical.Dockerfile

```diff
@@ -73,7 +73,7 @@ RUN cd /home/graphscope/GraphScope/ && \
         strip ${INSTALL_DIR}/bin/grape_engine; \
         strip ${INSTALL_DIR}/lib/*.so; \
         sudo cp -rs ${INSTALL_DIR}/* ${GRAPHSCOPE_HOME}/; \
-        python3 ./k8s/utils/precompile.py --graph --output_dir ${INSTALL_DIR}/builtin; \
+        python3 ./k8s/utils/precompile.py --graph --output_dir ${INSTALL_DIR}/builtin --enable_java_sdk ON; \
```

Comment:
is this supposed to be in analytical-java.Dockerfile?

Reply:
Currently there is no `analytical-java.Dockerfile`. Should we create one? @siyuan0322

Reply:
Ohh this is in the analytical-java part. My fault.